### ISSUE TITLE
Latest language details

### ISSUE THREAD

**Issue by Gonzalo2310** [sdras/array-explorer#34]
The line code numbers are approximate
src/App.vue
line 7:  'Array Explorer' Although this text could be the name of the application. In that case it should not be translated

line 8: 'Find the array method you need without digging through the docs'
line 12: 'Usage'

src/components/AppCode.vue
line 14: 'Output' 

src/components:MethodBase.vue
line 5: 'see the docs'

It's not important or urgent. It's just details to leave it 100% translated.

**Comment by sdras** [comment]
Yep, just a couple more PRs for the languages that are half done and we can build this out. I want to get it up and in first :)

Thanks for tracking it!
